src/Implementation/String/DateTimeValue.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ADS\ValueObjects\Implementation\String;
+
+use DateTime;
+use DateTimeInterface;
+
+use function strtotime;
+
+abstract class DateTimeValue implements \ADS\ValueObjects\DateTimeValue
+{
+    protected DateTimeInterface $value;
+
+    protected function __construct(DateTimeInterface $value)
+    {
+        $this->value = $value;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function fromString(string $value)
+    {
+        return new static(new DateTime('@' . strtotime($value)));
+    }
+
+    public function toString(): string
+    {
+        return $this->toDateTime()->format(self::getFormat());
```

Comment:
I would remove the getFormat string and use by default the RFC3339_EXTENDED. And add a function toFormattedString where you can give the format as a parameter.

Reply:
Don't forget to add this formatted function to the interface.

Reply:
Done